presidio-analyzer/analyzer/pattern_recognizer.py

```diff
@@ -12,8 +14,14 @@
 
 class PatternRecognizer(LocalRecognizer):
 
-    def __init__(self, supported_entity, name=None, supported_language='en',
-                 patterns=None,
+    CONTEXT_SIMILARITY_THRESHOLD = 0.65
+    CONTEXT_SIMILARITY_FACTOR = 0.35
+    MIN_SCORE_WITH_CONTEXT_SIMILARITY = 0.6
+    CONTEXT_PREFIX_COUNT = 5
+    CONTEXT_SUFFIX_COUNT = 0
```

Comment:
agree, I am opening a new task to fix this. the fix should be with a wider scope, so that even custom recognizers will be able to define their own values.
this means we also need to support it in the protobuf files.
its a wider task.